deepcell/utils/transform_utils.py

```diff
@@ -55,86 +55,70 @@ def pixelwise_transform(mask, dilation_radius=None, data_format=None,
-            for i in range(edges.shape[0]):
-                edges[i] = binary_dilation(edges[i], selem=dil_strel)
+            edge = binary_dilation(edge, selem=dil_strel)
 
             # Thin the augmented edges by subtracting the interior features.
-            edges = (edges - interiors > 0).astype('int')
+            edge = (edge - interior > 0).astype('int')
 
-        background = (1 - edges - interiors > 0)
+        background = (1 - edge - interior > 0)
         background = background.astype('int')
 
         all_stacks = [
-            edges,
-            interiors,
+            edge,
+            interior,
             background
         ]
 
-        return np.stack(all_stacks, axis=channel_axis)
+        return np.stack(all_stacks, axis=-1)
 
     # dilate the background masks and subtract from all edges for background-edges
-    dilated_background = np.zeros(mask.shape)
-    for i in range(mask.shape[0]):
-        background = (mask[i] == 0).astype('int')
-        dilated_background[i] = binary_dilation(background, strel)
+    background = (mask == 0).astype('int')
+    dilated_background = binary_dilation(background, strel)
 
-    background_edges = (edges - dilated_background > 0).astype('int')
+    background_edge = (edge - dilated_background > 0).astype('int')
 
     # edges that are not background-edges are interior-edges
-    interior_edges = (edges - background_edges > 0).astype('int')
+    interior_edge = (edge - background_edge > 0).astype('int')
 
     if dilation_radius:
-        dil_strel = ball(dilation_radius) if mask.ndim > 3 else disk(dilation_radius)
+        dil_strel = ball(dilation_radius) if mask.ndim > 2 else disk(dilation_radius)
         # Thicken cell edges to be more pronounced
-        for i in range(edges.shape[0]):
-            interior_edges[i] = binary_dilation(interior_edges[i], selem=dil_strel)
-            background_edges[i] = binary_dilation(background_edges[i], selem=dil_strel)
+        interior_edge = binary_dilation(interior_edge, selem=dil_strel)
+        background_edge = binary_dilation(background_edge, selem=dil_strel)
 
         # Thin the augmented edges by subtracting the interior features.
-        interior_edges = (interior_edges - interiors > 0).astype('int')
-        background_edges = (background_edges - interiors > 0).astype('int')
+        interior_edge = (interior_edge - interior > 0).astype('int')
+        background_edge = (background_edge - interior > 0).astype('int')
 
-    background = (1 - background_edges - interior_edges - interiors > 0)
+    background = (1 - background_edge - interior_edge - interior > 0)
     background = background.astype('int')
 
     all_stacks = [
-        background_edges,
-        interior_edges,
-        interiors,
+        background_edge,
+        interior_edge,
+        interior,
         background
     ]
 
-    return np.stack(all_stacks, axis=channel_axis)
+    return np.stack(all_stacks, axis=-1)
```

Comment:
Again, why not leave it as `channel_axis`?

Reply:
I removed the channel_axis argument to mirror the logic of the watershed transform, where it's evaluated within the image_generators __init__.py but then not passed into the function: https://github.com/vanvalenlab/deepcell-tf/blob/b78caac6263d46ef1d1570110cb647df0a5a03ed/deepcell/image_generators/__init__.py#L115

If you'd rather not make that same change here happy to revert back